NOTES.md

```diff
@@ -0,0 +1,60 @@
+# Notes from review
+
+* WebKit's documentation and intro on parking lot and its paradigm is quite good
```

Comment:
This https://webkit.org/blog/6161/locking-in-webkit/ ?

Reply:
Yes